src/pages/dar_application/ProgressReportApplication.tsx

```diff
@@ -39,7 +44,7 @@ export const ProgressReportApplication = ({ dar, datasets, readOnlyMode = true }
         ...(dar?.closeOutSupplement && {
             closeoutYesNo: !!dar.closeOutSupplement
         }),
-    }
+    };
 
     const [formState, setFormState] = useState<FormState>(initialState);
```

Comment:
The linter in my IDE is complaining about this because the initial state is actually the union of FormState and DataAccessRequest types.